zebrad/src/components/mempool/storage/tests/prop.rs

```diff
@@ -367,7 +367,8 @@ proptest! {
         let inserted_transactions: HashSet<_> = input.transactions().filter_map(|transaction| {
             let id = transaction.transaction.id;
 
-            storage.insert(transaction.clone()).ok().map(|_| id)}).collect();
+            storage.insert(transaction.clone()).ok().map(|_| id)
+        }).collect();
```

Comment:
Nitpick: I'm surprised rustfmt accepts this, do we want to revert this whitespace change?

Reply:
It seems rustfmt doesn't check the prop tests at all, I moved the code elsewhere to be formatted and updated it in [`51b841b`](https://github.com/ZcashFoundation/zebra/pull/5434/commits/51b841b6d98289211ec1f259d67922e5ed20c6e8).

Reply:
Yea, the test is in the `proptest!` macro, and rustfmt doesn't check code passed to macros. I wonder if there's ever going to be any support at all since code passed to macros doesn't have to follow Rust's syntax.